src/Maker/Helper/GridHelper.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of AppName.
+ *
+ * (c) Monofony
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Maker\Helper;
+
+class GridHelper
```

Comment:
I would still recommend `final` + `interface` on helper classes

Reply:
yes exactly.

Reply:
For the state, it's not publishable yet, I've not finished actions and yes filtration is missing.